docs/howto/operator.rst

```diff
@@ -0,0 +1,86 @@
+Using Operators
+===============
+
+An operator represents a single, ideally idempotent, task. Operators
+determine what actually executes when your DAG runs.
+
+See the :ref:`Operators Concepts <concepts-operators>` documentation and the
+:ref:`Operators API Reference <api-reference-operators>`for more information.
```

Comment:
Looks like this ref is wrong - screenshot shows it's not being turned in to a link

Reply:
Oops, missed some whitespace. Good catch.

Reply:
Fixed.

![image](https://user-images.githubusercontent.com/247555/42046784-7bd3beca-7ab3-11e8-8929-f897e6f59611.png)